Test.c

```diff
@@ -4,11 +4,141 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Test.c
-// Program on branch main
+// Program on branch A5
 // ------------------------------------------------------------------------------------------------
-
+#pragma warning(disable:4996)
+#include "Header.h"
 #include <stdio.h>
+#include <stdbool.h>
+#include <malloc.h>
+#include <string.h>
+#include <stdlib.h>
+
+typedef struct {
+   int error;            // Stores the error code.
+   int num;              // Stores the input value.
+}Type;
+
+/// <summary>Windows OS specific command to clear console screen.</summary>
+static void ClearScreen () {
+#ifdef _WIN32
+   system ("cls");
+#endif
+   while (getchar () != '\n');
+}
+
+/// <summary>Checks if the array is sorted in ascending order.</summary>
+bool IsSorted (int arr[], int length) {
+   for (int i = 1; i < length; i++) if (arr[i - 1] > arr[i]) return false;
+   return true;
+}
+
+void PrintArray (int arr[], int size, int expLength, char* str) {
+   printf (YELLOW" %s\t"RESET"|", str);
+   for (int i = 0; i < size; i++) printf ("%12d", arr[i]);
+   for (int i = expLength - size; i > 0; i--) printf ("%12s", "");
+}
+
+void TestHeapSort () {
+   int arr[8][8] = { { 1,2,3,5,4,6,7,8 } , { 8,7,6,5,4,3,2,1 } ,

```

Comment:
Add Test case for binary search, check with any element in the correct index after sorting